apps/graph-api/src/controllers/v1/webhooks-v1.controller.ts

```diff
@@ -51,8 +52,8 @@ export class WebhooksControllerV1 {
     required: false,
     description: "Boolean whether to include webhooks registered for 'all' MSA Ids (default: true)",
   })
-  async getWebhooksForMsa(@Param('msaId') msaId: string, @Query('includeAll') includeAll = 'true'): Promise<string[]> {
-    return this.apiService.getWebhooksForMsa(msaId, JSON.parse(includeAll ?? 'true'));
+  async getWebhooksForMsa(@Param() msaId: MsaIdParam, @Query('includeAll') includeAll = 'true'): Promise<string[]> {
```

Comment:
I forget how we standardized parameter naming, but I _think_ we landed on suffixing REST input parameter object types with `Dto` or `DTO`.

Also, I would like to try and standardize on documenting the DTO classes with `@ApiProperty()` decorators, which eliminates the need to document each endpoint with `@ApiParameter()` or `@ApiQuery()`